src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -79,10 +94,14 @@ export const NavBarSchedule: FC<NavBarScheduleProps> = ({ isVisible }) => {
   const [selectedDayIndex, setSelectedDayIndex] = useState(0);
 
   const weekdays = useMemo(() => {
-    const today = startOfToday();
+    const formatDayLabel = (day: Date) => {
+      if (isToday(day)) return "Today";
+      if (isToday(firstDayOfSchedule)) return format(day, "E");
+      return format(day, "E, LLL d");
+    };
```

Comment:
I'm not sure the logic here is correct, is it?

---

Related to outstanding question in https://github.com/sparkletown/sparkle/pull/1419#discussion_r638459361:

> ![image](https://user-images.githubusercontent.com/753891/119635959-85bb2a00-be57-11eb-8ac4-4d5b022deb61.png)
>
> ![image](https://user-images.githubusercontent.com/753891/119635877-7340f080-be57-11eb-8f50-70b0a5e4fb1f.png)
>
> Actually, looking at the code suggestion above (+ in https://github.com/sparkletown/sparkle/pull/1419/files#r635716283) vs what is implemented in https://github.com/sparkletown/sparkle/pull/1419/commits/b78373170c83bbc19324c8694b04ea35b9ab06ef, it looks like you didn't end up going with the `isScheduleTimeshifted` implementation.
> 
> Why was that?
> 
> _Originally posted by @0xdevalias in https://github.com/sparkletown/sparkle/pull/1419#discussion_r638459361_
>

Reply:
```suggestion
    const isScheduleTimeshifted = !isToday(firstDayOfSchedule);
```

Reply:
brought back in `isScheduledTimeshrifted`. done